<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formClass</class>
 <widget class="QMainWindow" name="formClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Собрать все XLSX в один в той же папке</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonCollect">
    <property name="geometry">
     <rect>
      <x>489</x>
      <y>360</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Собрать в один</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLoadFiles">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить файлы</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>491</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineLeftCol">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>331</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>A1:A10</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>581</width>
      <height>171</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>579</width>
       <height>169</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonPath">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать путь</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Путь к папке с xlsx файлами:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Крайний левый столбец:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Столбец для добавления в сводную таблицу:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineCol">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>331</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>B1:B10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя выходного файла:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineOut">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>441</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>1.xlsx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер листа, начиная с 0:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineSheet">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>331</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Внимание! Программа не умеет читать файлы, которые
сама сохранила! Убирайте напротив них галочки!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="form.qrc"/>
 </resources>
 <connections/>
</ui>
